src/main/java/bio/overture/ego/service/RefreshTokenService.java

```diff
@@ -27,18 +31,37 @@
   private RefreshTokenRepository refreshTokenRepository;
 
   /** Configuration */
-  @Value("${refreshToken.duration:10800000}")
-  private int DURATION;
+  private int durationSeconds;
 
   @Autowired
   public RefreshTokenService(
-    @NonNull RefreshTokenRepository refreshTokenRepository,
-    @NonNull UserRepository userRepository,
-    @NonNull UserService userService) {
+      @NonNull RefreshTokenRepository refreshTokenRepository,
+      @NonNull UserRepository userRepository,
+      @NonNull UserService userService,
+      @Value("${refreshToken.durationSeconds:10800000}") int durationSeconds
+  ) {
     super(RefreshToken.class, refreshTokenRepository);
+    this.durationSeconds = durationSeconds;
     this.refreshTokenRepository = refreshTokenRepository;
   }
 
+  //TODO: Ann to implement this
+  private RefreshToken  createTransient(User user){
+    val nowInstant = Instant.now();
+   val expiryInstant = nowInstant.plus(durationSeconds, SECONDS);
+    return RefreshToken.builder()
+        .jti(UUID.randomUUID()) //TODO [ANN] to implement propertly with jwt
+        .issueDate(Date.from(nowInstant))
```

Comment:
are `issueDate` and `expiryDate` moved here instead of the db to make them easier to manage?